### DISCUSSION TITLE
Strange progressbar on zipping big file

### DISCUSSION THREAD

**Discussion by Alexey-T** [doublecmd/doublecmd#1202] [General]
Make 3Gb file by python script
```
f=open('/home/user/big.txt', 'w')
for i in range(3*1024*1024*1024//100):
    f.write('e'*99+'\n')
del f
```

now use DC, to pack this txt file to .zip. Using 'store' zip method (option in the packer settings).

progressbar shows '4 B / 100 B'. bad.

![zip](https://github.com/doublecmd/doublecmd/assets/4154978/6b657785-b670-45a4-a09c-410989dd0efe)

DC 1.0.11~272, Xubuntu 22.

**Comment by alexx2000** [comment]
Yes, progress is not entirely correct in this case. Really it show percentages instead bytes.
